src/lib/components/organisms/Hero.svelte

```diff
@@ -27,6 +28,12 @@
 				Get Started
 			</Button>
 		{/if}
+		<Button href="http://index.torrust-demo.com">
```

Comment:
Hi @grmbyrn it should be: https://index.torrust-demo.com

Reply:
Hi @josecelano, I made the change from http to https.